src/parser.jl

```diff
@@ -214,7 +214,7 @@ end
 
 
 function parse_document_content(stream::EventStream)
-    if in(peek(stream.input), [DirectiveToken, DocumentStartToken, DocumentEndToken,StreamEndToken])
+    if peek(stream.input) isa Union{DirectiveToken, DocumentStartToken, DocumentEndToken,StreamEndToken}
```

Comment:
```suggestion
    if peek(stream.input) isa Union{DirectiveToken, DocumentStartToken, DocumentEndToken, StreamEndToken}
```

Might as well fix this bit of formatting at the same time